### ISSUE TITLE
In app player shows chapter times, not overall

### ISSUE THREAD

**Issue by pax0707** [prologueapp/Prologue#40]
In app player shows chapter times, not overall.
Works fine on the lock screen.

**Comment by lcharlick** [comment]
Yes, there's a setting for this under Settings > Playback > Playback Time, but it doesn't do anything yet.

**Comment by lcharlick** [comment]
Added in build 10455
